CHANGES/10146.misc.rst

```diff
@@ -0,0 +1 @@
+Setting :attr:`aiohttp.web.StreamResponse.last_modified` to an unsupported type will now raise :exc:`ValueError` instead of silently failing -- by :user:`bdraco`.
```

Comment:
```suggestion
Setting :attr:`aiohttp.web.StreamResponse.last_modified` to an unsupported type will now raise :exc:`TypeError` instead of silently failing -- by :user:`bdraco`.
```